src/sumo_store_mysql.erl

```diff
@@ -483,3 +484,28 @@ hash(Clause) ->
   List = binary_to_list(Bin),
   Fun = fun(Num) -> string:right(integer_to_list(Num, 16), 2, $0) end,
   lists:flatmap(Fun, List).
+
+%% @private
+wakeup(Doc) ->
+  Fields = sumo_store_utils:fields_from_doc(Doc),
+  lists:foldl(fun({FieldName, FieldType, FieldValue}, Acc) ->
```

Comment:
This code is repeated in multiple stores. Can we move it to sumo_utils or even better write code that is specific to each store (i.e. Not calling to_int when we know the backend returns integers already)?